scripts/client.js

```diff
@@ -1,47 +1,72 @@
+/* @flow */
+
 import KintoClient from "kinto-client";
 import endpoint from "kinto-client/lib/endpoint";
 import { isHTTPok } from "./utils";
 
 
-let client;
+let client: ?KintoClient;
 
-function getAuthHeader(session) {
-  const {authType, credentials} = session;
+function getAuthHeader(session: Object): ?string {
+  const {authType, credentials}: {
+    authType: string,
+    credentials: Object
+  } = session;
   switch(authType) {
     case "fxa": {
-      const {token} = credentials;
+      const {token}: {token: string} = credentials;
```

Comment:
Sadly that's the only type declaration syntax supported by flow for typed object destructured assignments.